.github/workflows/windows.yml

```diff
@@ -15,3 +15,38 @@ jobs:
         cd build
         cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_VERBOSE_MAKEFILE=ON -DAMReX_BUILD_TUTORIALS=ON -DAMReX_FORTRAN=OFF -DAMReX_MPI=OFF
         cmake --build . --config Debug
+
+  # Build libamrex and all tutorials with CUDA
+  tutorials-cuda:
+    name: MSVC C++17 w/ CUDA@11.1.1 w/o Fortran w/o MPI
+    runs-on: windows-2019
+    steps:
+    - uses: actions/checkout@v2
+    - name: Windows CUDA Cache
+      uses: actions/cache@v2
+      with:
+        path: "C:/Program Files/NVIDIA GPU Computing Toolkit/"
+        key: msvc_cxx17_cuda111
+    - name: Download CUDA
+      run: |
+        if (Test-Path "C:\Program Files\NVIDIA GPU Computing Toolkit\") { exit }
+        New-item -ItemType directory -Name cuda
+        Set-Location -Path cuda -PassThru
+        Invoke-WebRequest -Uri 'https://developer.download.nvidia.com/compute/cuda/11.1.1/network_installers/cuda_11.1.1_win10_network.exe' -OutFile 'cuda_install.exe'
+        Start-Process -FilePath '.\cuda_install.exe' -ArgumentList '-s' -Wait -NoNewWindow
+    - name: Build & Install
+      shell: bash
+      run: |
+        export CUDA_PATH="C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.1"
+        export CUDA_HOME=$CUDA_PATH
+        export PATH=$CUDA_PATH/bin:$PATH
+        export PATH="C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.28.29333/bin/Hostx64/x64/:$PATH"
+        export CUDACXX="$CUDA_PATH/bin/nvcc.exe"
+        export CC="C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.28.29333/bin/Hostx64/x64/cl.exe"
+        export CXX="C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.28.29333/bin/Hostx64/x64/cl.exe"
+        mkdir build
+        cd build
+        cmake -G "Visual Studio 16 2019" -A x64 -DAMReX_BUILD_TUTORIALS=ON -DAMReX_FORTRAN=OFF -DAMReX_MPI=OFF -DAMReX_GPU_BACKEND=CUDA -DCMAKE_CXX_STANDARD=17 -DCMAKE_CUDA_STANDARD=17 -DAMReX_CUDA_ARCH=6.0 -DCUDAToolkit_ROOT="C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.1" ..
+        cmake --build . --config Release
+    # use "cmd", see https://gitlab.kitware.com/cmake/cmake/-/issues/20281
+    # -DAMReX_PARTICLES=ON
```

Comment:
@mic84 as soon as I activated particles, it tried to search for a Fortran compiler. Maybe something to check later on.

Reply:
Maybe outdated, will check again once PR works.